Manual/Introduction to Inweb.w

```diff
@@ -38,38 +38,47 @@ it. Here's what to do. From a command-line prompt, set the current working
 directory to be the one in which Inweb is stored - that is, not the |inweb|
 directory itself, but its parent. Then type one of the following:
 = (text as ConsoleText)
-	$ make -f inweb/inweb.mk macos
 
-	$ make -f inweb/inweb.mk macos32
+	$ bash inweb/scripts/first.sh Linux
```

Comment:
I think this is supposed to be `linux` (and case sensitive)

Reply:
Updated to change case to `linux`